### PR TITLE
SR-035 App Settings Feedback [LB, MH]

### PR DESCRIPTION
**Pull request by marha333**
#3 Implemented Feedback in Settings [LB, MH]

### CODE REVIEW

File: app/src/main/java/at/team30/setroute/Helper/EmailHelper.kt [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+                }
+            })
+
+            session.debug = true
+
+            try {
+                val mimeMessage = MimeMessage(session)
+                mimeMessage.setFrom(InternetAddress(emailFrom))
+                mimeMessage.setRecipients(Message.RecipientType.TO, InternetAddress.parse(receiver, false))
+                //mimeMessage.setRecipients(Message.RecipientType.CC, InternetAddress.parse(emailCC, false))
```
Comment (jerlacher) [inline app/src/main/java/at/team30/setroute/Helper/EmailHelper.kt]: would remove commented lines

File: app/src/main/res/layout/fragment_settings.xml [hunk 2]
```diff
@@ -47,4 +48,71 @@
...
+<!--            android:layout_height="250dp"-->
+<!--            android:layout_margin="20dp"-->
+<!--            android:ems="10"-->
+<!--            android:hint="Type here..."-->
+<!--            android:inputType="textPersonName"-->
+<!--            android:maxLength="500"-->
+<!--            android:textAlignment="textStart"-->
+<!--            android:gravity="start"-->
+<!--            android:padding="10dp"-->
+<!--            />-->
```
Comment (jerlacher) [inline app/src/main/res/layout/fragment_settings.xml]: would remove unused code

**Review by jerlacher — APPROVED** [review]
Would suggest to remove the unused parts. Looks good :)

File: app/src/main/java/at/team30/setroute/Helper/EmailHelper.kt (hunk 1 above)
Comment (marha333) [reply app/src/main/java/at/team30/setroute/Helper/EmailHelper.kt]: Yeah you are right

**Comment by jerlacher** [comment]
Just saw that the test pipeline does not really work.

It seems like the Emulator has problems to start up correctly. I would suggest trying it again later on.
It might just be an issue with github.

**Pull request merged**